docs/03-dev-howtos/18-apply-the-special-report-tone.md

```diff
@@ -1,5 +1,11 @@
 # Apply the special report tone
 
+The Special Report tone can be added by central production by adding the required tag to the Special Tag Test Campaign in the Targeting Tool.
+
+## DEPRECATED 
+
+*The following method is deprecated and replaced with the targeting tool method, but kept here for prosperity as existing tags still use the method.*
```

Comment:
"posterity"?

Reply:
😆 yeah

Reply:
i'm gonna leave it like that for prosperity